### DISCUSSION TITLE
Should we release 0.70.10?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#66] [Patches]
## Released, please refer to #71

Current Release: [0.70.9](https://github.com/facebook/react-native/releases/tag/v0.70.9)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/03de19745eec9a0d4d1075bac48639ecf1d41352
2. https://github.com/facebook/react-native/commit/61106ac6805cddef97e16e473b155abdad701797
3. https://github.com/facebook/react-native/commit/a98da322293e2db777bcc599b5abb10f750a7816
4. https://github.com/facebook/react-native/commit/e0a71fc7b5cb8c264000147099be5b2575931193
5. https://github.com/facebook/react-native/pull/37452
6. https://github.com/facebook/react-native/commit/c05d822f7daa92e8af2ec2cd97a9897425624cc2
7. https://github.com/facebook/react-native/commit/10d55888cce0b625c4b05a19ed609154893d17d0

#### Local commits to backport to main

1.

**Comment by dmytrorykun** [comment]
List of commits that fix Hermes bytecode and source maps output. See https://github.com/facebook/react-native/issues/37419
https://github.com/facebook/react-native/commit/03de19745eec9a0d4d1075bac48639ecf1d41352
https://github.com/facebook/react-native/commit/61106ac6805cddef97e16e473b155abdad701797
https://github.com/facebook/react-native/commit/a98da322293e2db777bcc599b5abb10f750a7816
https://github.com/facebook/react-native/commit/e0a71fc7b5cb8c264000147099be5b2575931193

**Reply by joe-sam** [reply]
If use case exists then please fix this for android too.

**Reply by yhkaplan** [reply]
I confirmed it wasn't an issue with Android; the bundle in Android is Hermes bytecode.

**Comment by yhkaplan** [comment]
Note, profiling locally with Hermes bytecode and without it, using Hermes bytecode on an iPhone resulted in half the memory usage in my app, so this would patch a serious performance regression

**Comment by kelset** [comment]
https://github.com/facebook/react-native/pull/37452

**Comment by yhkaplan** [comment]
@kelset Thanks for handling this. Are there any blockers left for this release? Seems like a major regression and therefore high priority.

**Reply by kelset** [reply]
not sure, but now that 72RC5 is out we should have time to handle new releases for 71 and 70

**Comment by techgig17** [comment]
When is the planned release for this patch. We are blocked since iOS bytecode bundles are not generated. Please help us with this patch release

**Comment by malikzype** [comment]
https://github.com/facebook/react-native/pull/37487 This is already merged. Can we please take this issue?
Merge PR Link - https://github.com/facebook/react-native/commit/c05d822f7daa92e8af2ec2cd97a9897425624cc2

**Comment by cipolleschi** [comment]
We will need this for Xcode15 
- https://github.com/facebook/react-native/pull/37758

**Comment by tido64** [comment]
Just found that Xcode defaults to 'Release' when running `react-native run-ios` under specific setups. Fix here: https://github.com/react-native-community/cli/pull/1957. Would be great if we could get this into 0.71.

**Reply by kelset** [reply]
we're in the process of releasing 0.70.10 - it will need to be done in 0.70.11 once the right bump of CLI will be out

**Comment by robhogan** [comment]
Nominate this one for the RN-side fix to https://github.com/facebook/react-native/issues/36794 (to accompany a bump to Metro 0.72.4).
 - https://github.com/facebook/react-native/pull/37501 (https://github.com/facebook/react-native/commit/671ea383fe45dd9834a0c0481360de050df7f0c9)

**Reply by kelset** [reply]
we're in the process of releasing 0.70.10 - it will need to be done in 0.70.11 once the right bump of Metro/CLI will be out.
